applications/concept-catalogue/src/main/kotlin/no/begrepskatalog/conceptregistration/utils/BegrepPatch.kt

```diff
@@ -0,0 +1,23 @@
+package no.begrepskatalog.conceptregistration.utils
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import no.begrepskatalog.generated.model.Begrep
+import no.begrepskatalog.generated.model.JsonPatchOperation
+import java.io.StringReader
+import javax.json.Json
+import javax.json.JsonException
+
+@Throws(JsonException::class)
+fun patchBegrep(begrep: Begrep, operations: List<JsonPatchOperation>): Begrep {
+    if (operations.isNotEmpty()) {
+        val mapper = ObjectMapper().registerModule(JavaTimeModule())

```

Comment:
Hvorfor trenger vi JavaTimeModule?

Reply:
Slik at datoene serialiseres/deserialiseres korrekt.

Ellers får jeg en slik exception:
com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Cannot construct instance of `java.time.LocalDate`